identities/models.py

```diff
@@ -44,3 +44,11 @@ class Identity(models.Model):
 
     def __str__(self):
         return str(self.id)
+
+
+class OptOut(models.Model):
+    """An opt_out"""
```

Comment:
do you mean that an optout instance should have it's own explicit unique ID or that it should explicitly store the unique ID for the identity that the request pertains to?